packages/toolkit/config/environments.default.json

```diff
@@ -1,8 +1,8 @@
 {
-  "aragon:local": {
```

Comment:
As the default configuration is the one we fallback when we are not on an aragon project repo, names could be the same.